src/main/kotlin/dev/cbeck/tags/TagServer.kt

```diff
@@ -0,0 +1,41 @@
+package dev.cbeck.tags
+
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.google.inject.Guice
+import com.google.inject.Stage
+import dev.cbeck.tags.http.PBSerDeModule
+import dev.cbeck.tags.http.TagResource
+import io.dropwizard.Application
+import io.dropwizard.jdbi3.JdbiFactory
+import io.dropwizard.setup.Bootstrap
+import io.dropwizard.setup.Environment
+import java.lang.RuntimeException
+
+class TagServer : Application<TagConfiguration>() {
+    override fun initialize(bootstrap: Bootstrap<TagConfiguration>?) {
```

Comment:
When kotlin generates stubs for non-kotlin classes, they assume everything is optional (which seems reasonable), but if you know its not, you can remove the `?`. In this case, and down in `run()` on line 24, drop the optionals